.github/workflows/tests-docker.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         curl -X POST http://localhost/wis2downloader/subscriptions -H "Content-Type: application/json" -H "Authorization: Bearer github123" -d @test.json
     - name: populate stations from CSV 📡
       run: |
-        python3 wis2box-ctl.py execute wis2box metadata station publish-collection
+        python3 wis2box-ctl.py execute wis2box metadata station publish-collection --path /data/wis2box/metadata/station/station_list.csv
```

Comment:
can we split the station-list into separate files per test dataset, remove the step "populate stations from CSV" and instead add one bulk-insert command for each dataset being tested ?